### ISSUE TITLE
Update with --oldstyle please

### ISSUE THREAD

**Issue by orcutt989** [Vagdish/BGPyrtraits#1]

**Comment by Vagdish** [comment]
Heya,

I may need some directions here:
You mean by addind the --oldstyle argument, only two pictures are produced ? with the following sizes:
```
IMAGE_L_WIDTH_OLD = 110
IMAGE_L_HEIGHT_OLD = 170
IMAGE_S_WIDTH_OLD = 38
IMAGE_S_HEIGHT_OLD = 60
```

**Comment by orcutt989** [comment]
Yes! I saw you mentioned this functionality in the Beamdog forums, but
don't see it in action. Right now this is the only portrait making software
that exists so having the benefit to make portraits for both games would be
great. Thanks.
